pkg/api/v1/types.go

```diff
@@ -582,6 +582,8 @@ type RBDVolumeSource struct {
 	// Defaults to false.
 	// More info: http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it
 	ReadOnly bool `json:"readOnly,omitempty"`
+	// Optional: Sidecar is the sidecar container's name
+	Sidecar string `json:"sidecar,omitempty"`
```

Comment:
Sidecar is the image name. It is part of the pod API that user can specify so the appropriate container image can be started to execute the command.